rollup/internal/config/relayer.go

```diff
@@ -68,12 +95,22 @@ type RelayerConfig struct {
 	FinalizeBundleWithoutProofTimeoutSec uint64 `json:"finalize_bundle_without_proof_timeout_sec"`
 }
 
+// AlternativeGasTokenConfig The configuration for handling token exchange rates when updating the gas price oracle.
```

Comment:
lgtm

The one thing is maybe add here description which token symbol pair should be chosen. For example, if L1 gas token is `ETH` and L2 gas token `SCR`, which ticker should be used `ETHSCR` or `SCRETH`. This is not obvious from the first look

Reply:
thanks, added notation for this.